### PR TITLE
[AIRFLOW-XXX] Change default value for hive_tblproperties 

### PR DESCRIPTION
**Pull request by tyhan**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-XXX
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:
- AttributeError: 'NoneType' object has no attribute 'items'
- line: 110, file: operators/hive_to_druid.py

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - When adding new operators/hooks/sensors, the autoclass documentation generation needs to be added.
  - All the public functions and the classes in the PR contain docstrings that explain what it does

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

File: airflow/operators/hive_to_druid.py [hunk 1]
```diff
@@ -78,7 +78,7 @@ def __init__(
             target_partition_size=-1,
             query_granularity="NONE",
             segment_granularity="DAY",
-            hive_tblproperties=None,
+            hive_tblproperties=dict(),
```
Comment (mik-laj) [inline airflow/operators/hive_to_druid.py]: Look at: 
https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python

Comment (mik-laj) [reply airflow/operators/hive_to_druid.py]: Better idea is write:
```
        self.hive_tblproperties = hive_tblproperties or dict()
```
instead
````
        self.hive_tblproperties = hive_tblproperties
````

Comment (tyhan) [reply airflow/operators/hive_to_druid.py]: I modified it according to mik-laj's advice. 03252c86e446b3e81c0690c820b069177dc42a7d

**Comment by OmerJog** [comment]
This is a code change (bug fix) it needs a Jira ticket

**Comment by OmerJog** [comment]
This has been fixed in Master by https://github.com/apache/airflow/pull/5211

**Pull request closed without merging**